Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/manage-airflow-versions.md

> **Note:** If you'd like to change the version of Airflow you'd like to upgrade to, you can do so at anytime by clicking **Cancel**, re-selecting a new version and once again clicking **Upgrade**. More on that below.

> ### via the Astro CLI

Reviewer: ```suggestion
### with the Astro CLI
```

> To initialize the Airflow upgrade process via the Astronomer CLI, first make sure you're authenticated by running `$ astro login <base-domain>`.

Reviewer: ```suggestion
To use the Astro CLI to initialize the Airflow upgrade process, run `$ astro login <base-domain>` first to make sure you're authenticated .
```

### 1. Locate your Dockerfile in your Project Directory

> First, open the `Dockerfile` within your Astronomer directory. When you initialized an Airflow project via the Astro CLI, the following files should have been automatially generated:

Reviewer: ```suggestion
First, open the `Dockerfile` within your Astronomer directory. When you used the Astro CLI to initialize an Airflow project, the following files should have been automatically generated:
```
